config/flags.go

```diff
@@ -261,6 +261,7 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.String(StakingSignerKeyContentKey, "", "Specifies base64 encoded signer private key for staking")
 	fs.Bool(SybilProtectionEnabledKey, true, "Enables sybil protection. If enabled, Network TLS is required")
 	fs.Uint64(SybilProtectionDisabledWeightKey, 100, "Weight to provide to each peer when sybil protection is disabled")
+	fs.Bool(LiteSyncPrimaryNetworkKey, false, "Only sync the P-chain on the Primary Network. If the node is a Primary Network validator, it will shutdown")
```

Comment:
My only fear with this name is that people confuse it for a "state sync" or "pruning" mode. Maybe we should pick something more clear.

Reply:
I'd prefer `PartialSyncPrimaryNetwork`

Reply:
done